app/views/categories/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="d-flex flex-column align-items-center justify-content-center min-vh-100">
+  <h1 class="display-4 font-weight-bold" style="color: #434b54; font-weight: 600;">Budget App</h1>
+  <div class="d-flex flex-column align-items-center w-100 gap-2 mt-5">
+    <%= link_to "New? Register Here.", new_user_registration_path, class: 'btn btn-primary fs-5 text btn-lg w-75 badge py-4 mb-2' %>
+    <%= link_to "Registered? Sign In Now.", new_user_session_path, class: 'btn btn-primary fs-5 text btn-lg w-75 badge py-4' %>
+  </div>
+</div>
```

Comment:
- [ ] you have done a great design so far 👍  however, it would be better to follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), mostly on the *Colors and Layout: composition and space between elements* for the best user experience 🙏

**Template** | **Your design**
| --- | --- |
<img width="278" alt="Screen Shot 2023-10-20 at 22 28 49" src="https://github.com/kessie2862/Budget-App/assets/102757126/6fdc92a1-fa5b-4671-a6f9-2638d4252ab3"> | <img width="407" alt="Screen Shot 2023-10-20 at 22 30 25" src="https://github.com/kessie2862/Budget-App/assets/102757126/b0e8dd53-c7f6-481d-9ee5-56dc81bbaca7">

Reply:
I don't understand this feedback. Could you throw more light on it? 
The code you highlighted is not related to the transactions page image above.

Reply:
I have made the necessary changes. Thanks